src/WinGet.RestSource/Constants/ApiConstants.cs

```diff
@@ -35,9 +35,9 @@ public class ApiConstants
 
         /// <summary>
         /// Unsupported package match fields.
-        /// Note: NormalizedPackageNameAndPublisher field is added here to demonstrate how this property can be used.
+        /// Note: NormalizedPackageNameAndPublisher field support is currently not implemented. There is a GitHub issue for this.

```

Comment:
Link or issue # might be appropriate here

Reply:
Added the issue link